MediaBrowser.WebDashboard/dashboard-ui/css/librarybrowser.css

```diff
@@ -216,6 +216,7 @@
 
 .itemOverview {
     white-space: pre-wrap;
+    font-size: 20px;
```

Comment:
I understand. But still, this may need adjust, I believe.
I consider it a little too small, and the style doesn't control it's size at all, today.